owncloudApp/src/main/java/com/owncloud/android/presentation/ui/security/PassCodeActivity.kt

```diff
@@ -204,6 +214,13 @@ class PassCodeActivity : BaseActivity() {
                 /// pass code accepted in request, user is allowed to access the app
                 passCodeError.visibility = View.INVISIBLE
                 hideSoftKeyboard()
+                if (passCodeViewModel.getPassCode()!!.length < passCodeViewModel.getNumberOfPassCodeDigits()) {
+                    passCodeViewModel.removePassCode()
+                    val intent = Intent(baseContext, PassCodeActivity::class.java)
```

Comment:
This would be more Kotlin's friendly:
`intent.apply{
action = ACTION_REQUEST_WITH_RESULT
putExtra(EXTRAS_MIGRATION, true)
}`

Reply:
👍